src/clients/client.cpp

```diff
@@ -155,63 +137,34 @@ int main(int argc, char** argv)
     "Remote node JSON-RPC server address");
   app.add_option("--ca", ca_file, "Network CA", true);
 
-  auto member_rpc = app.add_subcommand("memberrpc", "Member RPC");
-
-  std::string req_mem = "@memberrpc.json";
-  std::string member_cert_file = "member1_cert.pem";
-  std::string member_pk_file = "member1_privk.pem";
-  add_request_arg(member_rpc, req_mem);
-  member_rpc->add_option(
-    "--cert", member_cert_file, "Member's certificate", true);
-  member_rpc->add_option("--pk", member_pk_file, "Member's private key", true);
-
-  auto user_rpc = app.add_subcommand("userrpc", "User RPC");
-
-  std::string req_user = "@userrpc.json";
-  std::string user_cert_file = "user1_cert.pem";
-  std::string user_pk_file = "user1_privk.pem";
-  add_request_arg(user_rpc, req_user);
-  user_rpc->add_option("--cert", user_cert_file, "User's certificate", true);
-  user_rpc->add_option("--pk", user_pk_file, "User's private key", true);
+  std::string req = "@rpc.json";
+  std::string client_cert_file;
+  std::string client_pk_file;
+  app.add_option("--req", req, "RPC request data, '@' allowed", true);
+  app.add_option("--cert", client_cert_file, "Client certificate", true);
+  app.add_option("--pk", client_pk_file, "Client private key", true);
 
   CLI11_PARSE(app, argc, argv);
 
   try
   {
-    host = server_address.hostname;
-    port = server_address.port;
-
     nlohmann::json response;
 
-    cout << fmt::format("Sending RPC to {}:{}", host, port) << endl;
-
-    if (*member_rpc)
-    {
-      cout << "Doing member RPC:" << endl;
-      response = make_rpc(
-        host,
-        port,
-        Pack::MsgPack,
-        "members",
-        ca_file,
-        member_cert_file,
-        member_pk_file,
-        req_mem);
-    }
-
-    if (*user_rpc)
-    {
-      cout << "Doing user RPC:" << endl;
-      response = make_rpc(
-        host,
-        port,
-        Pack::MsgPack,
-        "users",
-        ca_file,
-        user_cert_file,
-        user_pk_file,
-        req_user);
-    }
+    cout << fmt::format(
+              "Sending RPC to {}:{}",
+              server_address.hostname,
+              server_address.port)
+         << endl;
+
+    cout << "Doing RPC:" << endl;
```

Comment:
Let's delete that line, it only served to indicate which frontend the RPC was sent to, but we know that from the method now.